### ISSUE TITLE
Download Grid Folder as Archive

### ISSUE THREAD

**Issue by ngwnos** [mcmonkeyprojects/SwarmUI#208]
### Feature Idea

The "generate webpage" functionality of the grid generator is amazing. It would be nice to have an easy way to download the entire folder as something self-contained with all the folder structure and images as a zip archive.

**Comment by mcmonkey4eva** [comment]
While there should indeed be an interface feature for this, as long as you have access to the system swarm is running on you can just open up the output directory and zip it up manually at least

**Comment by ngwnos** [comment]
Yeah, it's definitely not urgent, but there are a lot of features that make it very convenient to run remotely with almost no setup other than opening up a port. Bulk downloading in general is kinda the only thing that feels like it's really missing there.

**Comment by shorties** [comment]
I too would like some easier way to download the files from either the simple, generate, or ComfyUI outputs, in bulk. When running in a situation like a runpod instance, it actually is kind of a hassle to get to the various output folders and to download bulk sets of images, at least through the main SwarmUI template they have on there with Jupyter notebooks being a main gui way to access the filesystem. Which doesnt support bulk downloads without installing an extension that I never have gotten to work (For some reason I also have had trouble with SSH/SFTP with that template) 

Most of this is outside of project as I know the template isnt part of this project. But I was just describing it as a illustration as to why it would be a very useful to have an easy way to browse and bulk download from the multiple output folders.
